src/Fixer/PhpBasic/Feature/VoidResultFixer.php

```diff
@@ -67,7 +76,7 @@ public function applyFix(SplFileInfo $file, Tokens $tokens)
                 $token->isGivenKind(T_STRING)
                 && $tokens[$key + 1]->equals('(')
                 && $tokens[$functionTokenIndex]->isGivenKind(T_FUNCTION)
-                && $tokens[$visibilityTokenIndex]->isGivenKind([T_PUBLIC, T_PROTECTED, T_PRIVATE])
+                && ($visibilityTokenIndex || $tokens[$visibilityTokenIndex]->isGivenKind([T_PUBLIC, T_PROTECTED, T_PRIVATE]))
```

Comment:
strict comparison should be used, also looks like logic is incorrect as condition `$tokens[$visibilityTokenIndex]` won't be checked if `$visibilityTokenIndex` is truthy, which is probably oposite of what should be achieved here